src/api/printable_result.py

```diff
@@ -13,6 +14,46 @@ def print(self):
         stringifier = ConsoleStringifier(c.configuration.to_stringifier_config())
         print(stringifier.result_to_str(self._result))
 
+    def get_str(self) -> str:
+        stringifier = ConsoleStringifier(c.configuration.to_stringifier_config())
+        return stringifier.create_str(self._result)
+
+    def get_str_value(self) -> str:
+        stringifier = ConsoleStringifier(c.configuration.to_stringifier_config())
+        return stringifier.create_str_value(self._result)
+
+    def get_str_without_uncert(self) -> str:
+        stringifier = ConsoleStringifier(c.configuration.to_stringifier_config())
+        return stringifier.create_str_without_uncert(self._result)
+
+    def get_uncerts(self) -> list[dict[str, str]]:
+        stringifier = ConsoleStringifier(c.configuration.to_stringifier_config())
+        return [
+            {
+                "name": u.name,
+                "string": stringifier.create_str_uncert(u, self._result.unit),
+            }
+            for u in self._result.uncertainties
+        ]
```

Comment:
Now IntelliSense can't help you with this anymore, it doesn't know that this returns an object that has keys `name` and `string`. Please use a **proper type** for this.

And another note: you didn't include just the **value of the uncertainty** as string here, right? I'd like to be able to do
```py
res = wiz.res(...)
print(res.uncerts[0].value)
```